pyramid_oereb/lib/records/view_service.py

```diff
@@ -66,16 +66,48 @@ class ViewServiceRecord(object):
     # Attributes defined while processing
     image = None    # map image resulting from calling the wms link - binary
 
-    def __init__(self, reference_wms, legend_at_web=None, legends=None):
+    def __init__(self, reference_wms, layer_index, layer_opacity,
+                 legend_at_web=None, legends=None,
+                 min_NS03=None, max_NS03=None,
+                 min_NS95=None, max_NS95=None):
         """
 
         Args:
             reference_wms (uri): The link URL to the actual service (WMS)
+            layer_index (int): Layer index. Value from -1000 to +1000.
+            layer_opacity (float): Opacity of layer. Value from 0.0 to 1.0.
             legend_at_web (uri): The link URL to the actual legend service (WMS get legend)
             legends (list of LegendEntry): A list of all relevant legend entries.
+            min_NS03 (list): TODO doc.
```

Comment:
@jwkaltz Can we find out, what these attributes are for?

Reply:
Could this be some kind of extent? (In 21781 and 2056)

Reply:
The new specification allows to have the data either in EPSG:21781 (NS03 = national survey 1903) or in EPSG:2056 thus the two attribute versions. Normally the geometries are provided in only one of the two SRS.

Reply:
and yes, it's the bbox extends.

Reply:
> and yes, it's the bbox extents.

Ah, that is what I wanted to know, thanks! :+1:

Reply:
just in case: 
page 5 of the new specifications, class Map
F: https://www.cadastre.ch/fr/manual-oereb/publication/instruction.detail.document.html/cadastre-internet/fr/documents/oereb-weisungen/OEREB-Data-Extract_fr.pdf.html
D: https://www.cadastre.ch/de/manual-oereb/publication/instruction.detail.document.html/cadastre-internet/de/documents/oereb-weisungen/OEREB-Data-Extract_de.pdf.html